addons/mass_mailing/tests/test_mailing_ui.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+from odoo.addons.base.tests.common import HttpCaseWithUserDemo
+
+
+@odoo.tests.tagged('-at_install', 'post_install')
+class TestUi(HttpCaseWithUserDemo):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env.ref('base.user_demo').groups_id |= cls.env.ref('mass_mailing.group_mass_mailing_user')
```

Comment:
Actually you should use the demo user defined in HttpCaseWithUserDemo (aka cls.user_demo) . That's the whole purpose of this class, being independent from existing users in DB .

Reply:
@tde-banana-odoo Done.